crowdin.yml

```
@@ -0,0 +1,79 @@
+# Project Public Identifier: joomla-official-extensions
+project_id_env: CROWDIN_PROJECT_ID # The numeric project ID.
+api_token_env: CROWDIN_PERSONAL_TOKEN
+base_path: ".."
+base_url: "https://joomla.crowdin.com"
+
+preserve_hierarchy: true
+
+files:
+  - source: /src/administrator/language/en-GB/en-GB.com_weblinks.ini
+    dest: 'Weblinks/administrator/language/en-GB.com_weblinks.ini'
+    translation: /administrator/language/%locale%/%locale%.com_weblinks.ini
```

```suggestion
    translation: /administrator/language/%locale%/com_weblinks.ini
```

de-DE.com_weblinks.ini is deprecated so I would say we can go with %locale%/com_weblinks.ini alone. There could be the argument made to also do the change (remove en-GB) to the source folders

---

That's why I left it with en-gb. 
All translators would have to create a script again and delete data.

---

Here its something that only the component has to ship so the individual transalators are not required to do the stuff as the files are shipped with the extension. But yes it works both ways.